src/cliques.jl

```diff
@@ -37,24 +40,32 @@ function pothen_sun{T<:Integer}(par::Vector{T}, post::Vector{T}, colcount::Vecto
     ns, flag
 end
 
-function chordal_embedding{Tv<:Number,Ti<:Int}(A::SparseMatrixCSC{Tv,Ti}, Perm::Vector{Ti})
+function round_array(T, A::SparseMatrixCSC{Tv,Ti}) where {Tv<:Number,Ti<:Int}
+    if T <: Integer
+        convert(SparseMatrixCSC{T,Ti},map(x->Base.round(T,x),A))
+    else 
+        convert(SparseMatrixCSC{T,Ti},map(x->Base.round(x),A))
+    end
+end
+
+function chordal_embedding(A::SparseMatrixCSC{Tv,Ti}, Perm::Vector{Ti}) where {Tv<:Number,Ti<:Int}
```

Comment:
Here there is a c function used which I don't recall. Therefore, I suggest changing the function. I will try to recode it.